src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -687,7 +687,12 @@ export default class DefaultDeviceController
       return;
     }
     let isChanged = false;
-    for (const track of this.activeDevices['audio'].stream.getTracks()) {
+    let audioStream = this.activeDevices['audio'].stream 
+    if (this.useWebAudio) {
```

Comment:
Do we need to add unit test to cover this if condition?

Reply:
We already have tests that trigger this condition